src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -288,12 +288,25 @@ def build_output_dir(output_dir: Path, subdir: Optional[Union[str, Path]] = None
     return output_dir
 
 
-def build_final_results_directory(results_dir: str) -> Tuple[Path, Path]:
+def build_final_results_directory(
+    results_dir: str,
+    version: Optional[str] = None,
+) -> Tuple[Path, Path]:
+    if version is None:
+        subdir = (
+            paths.FINAL_RESULTS_DIR_NAME
+            / datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
+            / "pseudopeople_input_data_usa"
+        )
+    else:
+        subdir = (
+            paths.FINAL_RESULTS_DIR_NAME
+            / datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
```

Comment:
@rmudambi Do we want the version number to replace this timestamp altogether? It would make it easier to find (at least until we fix the symlink generation).

Reply:
We are leaving it since we will update the symlink generation at some point.